Project3/styles.css

```
@@ -0,0 +1,111 @@
+input[type="radio"],
+input[type="checkbox"] {
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    appearance: none;
+    border: 2px solid white;
+    height: 18px;
+    width: 18px;
+    display: inline-block;
+}
+
+/* font-family: 'BenchNine', sans-serif; */
+/* all width 800px 1024px 1120px (form) */
+body {
+    background-color: #163346;
+    align: auto;
+    padding: 16px;
+    margin: auto;
+    max-width: 300px;
+    min-width: 300px;
+    font-family: 'BenchNine', sans-serif;
+}
+
+form {
+    background-color: #1e2019;
+    padding: 8px;
+    max-width: 250px;
+    min-width: 250px
+}
+
+h2,
+label,
+legend,
+p {
+    color: #fff;
+
+}
+
+h2 {
+    font-family: 'Teko', sans-serif;
+    font-weight: 500;
+    font-size: 35px;
+    line-height: auto;
+    text-align: left;
+    border-bottom: #163346 solid 3px;
+
+}
+
+label,
+legend,
+p {
+    font-family: 'BenchNine', sans-serif;
+    font-size: 18px;
+    font-weight: 800px;
+
+}
+
+.left-column {
+    float: left;
+    margin-right: 1px;
+    margin-bottom: 12px;
+    margin-top: 16px;
+    border: transparent;
+
+}
+
+.right-column {
+
+    margin-right: 1px;
+    margin-bottom: 12px;
+    border: transparent;
+
+}
+
+button-reset {
```

Missing the period to reference the classname here:
```suggestion
.button-reset {
```

Also, worth noting that one of the values in a reset class is that it is reusable; here, by making it more specific, you're losing that value. Instead of applying these styles to the reset class, you want to apply the to _either_ a new class name, applied to the button _in addition to_ the existing `button-reset` class **or** to the base `button[type="submit"]` element.